src/pages/PokemonList.tsx

```diff
@@ -0,0 +1,65 @@
+/** @format */
+import {
+	useContext,
+	useEffect,
+	useState,
+} from "react";
+import PokePagination from "../components/PokePagination";
+import ListItemCard from "../components/ListItemCard";
+import "./PokemonList.css";
+import Typography from "@mui/material/Typography";
+import { PokemonContext } from "../context/PokemonContext";
+import { Pokemon } from "../context/Domain";
+
+const PokemonList = () => {
+	const pokemonContext = useContext(
+		PokemonContext
+	);
+
+	const [
+		pokemonsList,
+		setPokemonsList,
+	] = useState<Pokemon[]>();
+	useEffect(() => {
+		const list =
+			pokemonContext?.pokemonResult
+				?.results;
+		setPokemonsList(list);
+	}, [pokemonContext?.pokemonResult]);
+
+	return (
+		<div className="Screen">
+			<img
+				src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/25.png"
+				alt="pikachu"
+				style={{
+					height: 200,
+					width: 200,
+				}}
+			/>
+			<div className="Sitename">
+				<Typography variant="h5">
+					Pokemon Explorer
+				</Typography>
+			</div>
+
```

Comment:
Somehow it is weird to see a mix of MUI and core html components

Reply:
Fixed